docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: "3.4"
+services:
+  conference-app:
+    container_name: conference-app
+    env_file:
+      - .env
+      - envs/conference-app.env
+    command:
+      - sh
+      - -c
+      - |
+        if [ -e tmp/pids/server.pid ]; then
+          rm tmp/pids/server.pid
+        fi
+        bundle install
+        bin/rails s -p 3000 -b 0.0.0.0
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - conference-app-vendor:/vendor
+    stdin_open: true
+    tty: true
+    depends_on:
+      - conference-app-yaichi
+      - conference-app-db
+      - conference-app-worker
+  conference-app-css:
+    extends:
+      service: conference-app
+    container_name: conference-app-css
+    command:
+      - sh
+      - -c
+      - |
+        bundle install
+        bin/rails tailwindcss:watch
+  conference-app-worker:
+    container_name: conference-app-worker
+    env_file:
+      - .env
+      - envs/conference-app.env
+    command:
+      - sh
+      - -c
+      - |
+        if [ -e tmp/pids/server.pid ]; then
+          rm tmp/pids/server.pid
+        fi
+        bundle install
+        bundle exec sidekiq --queue default
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - conference-app-vendor:/vendor
+    depends_on:
+      - conference-app-yaichi
+      - conference-app-db
+      - conference-app-redis
+    stdin_open: true
+    tty: true
+  conference-app-db:
+    container_name: conference-app-db
+    image: postgres:15.4-bookworm
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    volumes:
+      - conference-app-postgres-data:/var/lib/postgresql/data
+    ports:
+      - 15432:5432
+  conference-app-redis:
+    container_name: conference-app-redis
+    image: redis:7.2-bookworm
+    volumes:
+      - conference-app-redis-data:/data
+    ports:
+      - 16379:6379
+  conference-app-yaichi:
```

Comment:
I'm a little concerned about whether using [Yaichi](https://github.com/mtsmfm/yaichi) is best for this application.

I feel that the original behavior can be reproduced without using Yaichi.

Reply:
Thank you! You're right! It's now accessible from localhost:3000 after running
```sh
docker compose up conference-app
```

3b4a948

Reply:
I didn't know about yaichi. It's a pretty good product.
But I don't know if it's allowable or not to enable webpush on `foo.localhost`.
WebPush (service worker) is allowed in only localhost and https scheme domains. Using https in local env is a bit hard.

https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers